### PR TITLE
Update the release note with the standard lib changes

### PR DESCRIPTION
**Pull request by IrushiL**
## Purpose
$subject

Related PR: https://github.com/ballerina-platform/ballerina-lang/pull/29937

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-alpha5-release-note.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
 #### New Features
 
 #### Improvements
+`jballerina.java` Package
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha5-release-note.md:61]: Shouldn't this be
```suggestion
##### `jballerina.java` Package
```

or
```suggestion
##### Java Package
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 2]
```diff
@@ -58,6 +58,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha5-release-note.md:62]:
```suggestion
- Dependently-typed `java:cast` Function

```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 3]
```diff
@@ -58,6 +58,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+Improved the `cast` function to determine return type based on the inferred type.
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha5-release-note.md:63]:
```suggestion
The `java:cast` function is now a dependently-typed function. If the `typedesc` argument is not provided, it is inferred from the contextually-expected type.
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 4]
```diff
@@ -58,6 +58,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+```bal
+FileInputStream|error obj = java:cast(inputStream);
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha5-release-note.md:65]:
```suggestion
FileInputStream|error obj1 = java:cast(inputStream, FileInputStream);
FileInputStream|error obj2 = java:cast(inputStream); // The second argument is inferred to be `FileInputStream`.
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 5]
```diff
@@ -58,8 +58,38 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+
+The `java:cast` function is now a dependently-typed function. If the `typedesc` argument is not provided, it is inferred from the contextually-expected type.
+```bal
+FileInputStream|error obj1 = java:cast(inputStream, FileInputStream);
+FileInputStream|error obj2 = java:cast(inputStream); // The second argument is inferred to be `FileInputStream`.
+```
+
+##### File Package Updates
+
+- The `path` field in the `ListenerConfig` has changed from optional to mandatory.
```
Comment (praneesha) [inline release-notes/swan-lake-alpha5-release-note.md:72]:
```suggestion
- Changed the `path` field in the `ListenerConfig` from optional to mandatory.
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 6]
```diff
@@ -58,8 +58,38 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+- Improved the compiler plugin validation for the service.
+
+##### Log Package Updates
+
+- Changed the time format in log output to RFC3339
```
Comment (praneesha) [inline release-notes/swan-lake-alpha5-release-note.md:77]:
```suggestion
- Changed the time format in the log output to RFC3339.
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 7]
```diff
@@ -58,8 +58,38 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+- Added validation to check log levels in Config.toml file
```
Comment (praneesha) [inline release-notes/swan-lake-alpha5-release-note.md:78]:
```suggestion
- Added validation to check log levels in the `Config.toml` file.
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 8]
```diff
@@ -58,8 +58,38 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+
+##### Sql Package Updates
```
Comment (praneesha) [inline release-notes/swan-lake-alpha5-release-note.md:80]:
```suggestion
##### SQL Package Updates
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 9]
```diff
@@ -58,8 +58,38 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+If return types of the SQL time related data types are expected as records, then the type of those records are now validating and should be as follows,
```
Comment (praneesha) [inline release-notes/swan-lake-alpha5-release-note.md:81]:
```suggestion
If return types of the SQL time-related data types are expected as records, then the type of those records will be validated now and should be as follows.
```

File: release-notes/swan-lake-alpha5-release-note.md [hunk 10]
```diff
@@ -58,8 +58,38 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
...
+- DATE -> time:Date
+- TIME -> time:TimeOfDay
+- TIME WITH TIME ZONE -> time:TimeOfDay
+- TIMESTAMP -> time:Civil
+- TIMESTAMP WITH TIME ZONE -> time:Civil
 
 #### Bug Fixes
+##### Log Package Bug fixes
+
+Made log module escape characters properly (https://github.com/ballerina-platform/ballerina-standard-library/issues/1192).
```
Comment (praneesha) [inline release-notes/swan-lake-alpha5-release-note.md:92]:
```suggestion
Made log module escape characters properly (https://github.com/ballerina-platform/ballerina-standard-library/issues/1192).
```
It is not clear what has been done.

Comment (IrushiL) [reply release-notes/swan-lake-alpha5-release-note.md:92]: @daneshk

Comment (daneshk) [reply release-notes/swan-lake-alpha5-release-note.md:92]: Shall we change like `Invalid Characters in log message are escaped properly`

Comment (praneesha) [reply release-notes/swan-lake-alpha5-release-note.md:92]: @IrushiL - Please change to "Invalid characters are now escaped properly in log messages."

Comment (IrushiL) [reply release-notes/swan-lake-alpha5-release-note.md]: Done

**Review by praneesha — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**